sdk/storage/azure-storage-file-share/tests/test_file_async.py

```diff
@@ -914,7 +914,9 @@ def read(self, count):
     @AsyncStorageTestCase.await_prepared_test
     async def test_copy_file_async_private_file_async(self, resource_group, location, storage_account, storage_account_key, rmt, rmt_key):
         self._setup(storage_account, storage_account_key, rmt.name, rmt_key)
-        await self._create_remote_share()
```

Comment:
It seems create remote share was to create a share in another account, while this change is to test the share within the same account.